lambda/src/main/scala/pricemigrationengine/services/CohortTableTest.scala

```diff
@@ -26,7 +25,7 @@ object CohortTableTest {
             _ <- console.putStrLn(s"Updating cohort table with result: $result")
           } yield ()
 
-        def update(result: SalesforcePriceRiseCreationResult): ZIO[Any, CohortUpdateFailure, Unit] = ???
+        def update(subscriptionName: String, result: SalesforcePriceRiseCreationDetails): ZIO[Any, CohortUpdateFailure, Unit] = ???
```

Comment:
`subscriptionNumber` seems to be the most common name, but it's not consistent in Zuora so probably interchangeable.

Reply:
I was using subscriptionName based on that is what is used in the schema spreadsheet, we should probably make it consistent. Ill leave it as is we can come to a decision  and change it everywhere.